internal/cmd/skupper/token/kube/token_issue.go

```diff
@@ -70,9 +71,18 @@ func (cmd *CmdTokenIssue) ValidateInput(args []string) error {
 		if !ok {
 			validationErrors = append(validationErrors, fmt.Errorf("token file name is not valid: %s", err))
 		} else {
-			// check we can use as a filename
-			if _, err := os.ReadDir(args[0]); err == nil {
-				validationErrors = append(validationErrors, fmt.Errorf("token file name is a directory"))
+			// check filename and directory are valid to create the token
+			directory, filename := filepath.Split(args[0])
```

Comment:
Before doing it, it would be safe to run `os.Stat(args[0])` and ensure it is not a directory.
As is, if you pass a directory that is writable to the current user, it hangs then throw an
error saying grant is not ready yet.